nunit-project-editor.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-project-editor</id>
+    <version>1.0</version>
+    <packageSourceUrl>https://github.com/nunit/nunit-project-editor/chocolatey/</packageSourceUrl>
+    <owners>Charlie Poole</owners>
+
+    <title>NUnit Project Editor</title>
+    <authors>Charlie Poole</authors>
+    <projectUrl>https://github.com/nunit/nunit-project-editor</projectUrl>
+    <iconUrl>http://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_64.png</iconUrl>
+    <copyright>2016 Charlie Poole</copyright>
+    <licenseUrl>http://github.com/nunit/nunit-project-editor/LICENSE.txt</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/nunit/nunit-project-editor</projectSourceUrl>
+    <docsUrl>https://github.com/nunit/docs/wiki/Project-Editor</docsUrl>
```

Comment:
I can include them if you think it's helpful (it's optional as far as chocolatey.org is concerned).

Not sure whether to use a mailto: or the google groups url for support. You need to have a google id for the latter, so maybe the email is better.